storageservice/modules/files/utils.bal

```diff
@@ -82,22 +87,22 @@ isolated function createErrorFromXMLResponse(http:Response response) returns err
 
     match statusCode {
         http:STATUS_CONFLICT => {
-            return error ConflictError("Conflict occurred.", errorCode = errorCode, message = message);
+            return error ConflictError("Conflict occurred.", httpStatus = 409, errorCode = errorCode, message = message);
```

Comment:
Instead of hardcoding the statusCode, we can do like this
```suggestion
            return error ConflictError("Conflict occurred.", httpStatus = statusCode, errorCode = errorCode, message = message);
```